db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+user = User.create()
```

Comment:
Useless assignment to variable - `user`.<br>Do not use parentheses for method calls with no arguments.